docs/attributes-registry/k8s.md

```diff
@@ -0,0 +1,213 @@
+# Kubernetes
+
+**Status**: [Experimental][DocumentStatus]
+
+Useful resources to understand Kubernetes objects and metadata:
+
+* [Namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)
+* [Names and UIDs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/).
+* [Pods](https://kubernetes.io/docs/concepts/workloads/pods/)
+* [Controllers](https://kubernetes.io/docs/concepts/workloads/controllers/)
+
+The "name" of a Kubernetes object is unique for that type of object within a
```

Comment:
Do we need all of that description in the registry? It feels being redundant to what is described in `/resource/k8s.md`.

How about rendering a single, plain table here? And have the breakdown into groups plus additional descriptions in the  `/resource/k8s.md` file?